### DISCUSSION TITLE
I dont understand what is chekpoint_path and kmeans_path in wav2vec

### DISCUSSION THREAD

**Discussion by Boris-sudo** [lucidrains/musiclm-pytorch#60] [Q&A]
I have done like in example
```
wav2vec = HubertWithKmeans(
    checkpoint_path='./hubert/hubert_base_ls960.pt',
    kmeans_path='./hubert/hubert_base_ls960_L9_km500.bin'
)
```
But I got an error 
**AssertionError: path ./hubert/hubert_base_ls960.pt does not exist**
Which path should I pass there?

**Comment by gokulnotre5** [comment]
You need to download hubert checkpoints at https://github.com/facebookresearch/fairseq/tree/main/examples/hubert. After downloading you need to paste the path of files over here
